### ISSUE TITLE
Deploying multiple operator instances fails

### ISSUE THREAD

**Issue by abrodersen** [zalando/postgres-operator#875]
This appears to be caused by #832.

Deploying the first release of the operator using chart version `1.4.0` works. However, when deploying a second instance, helm returns this error:

```
Error: UPGRADE FAILED: kind ClusterRole with the name "postgres-pod" already exists in the cluster and wasn't defined in the previous release. Before upgrading, please either delete the resource from the cluster or remove it from the chart
```

I tried setting `rbac.create` to `false` but this causes other necessary RBAC resources to not get created. The chart also does not offer any way to customize the name of the `postgres-pod` ClusterRole. If this customization were available, multiple instances could be deployed without issue.

**Comment by FxKu** [comment]
The name of this cluster role should indeed be configurable. Actually the [`pod_service_account_name`](https://github.com/zalando/postgres-operator/blob/07c5da35e3572b8302f33e3dd8fbf5e8ca846229/charts/postgres-operator/values.yaml#L100) field is meant for that. Would it work for you, to change the `values.yaml` for each operator you are deploying? Or do you want a generated name from helm?

**Comment by abrodersen** [comment]
@FxKu thank you! It looks like your pull request will resolve the issue.

**Comment by FxKu** [comment]
@abrodersen we merged it. Thanks for raising this. By default the additional cluster role is still named [`postgres-pod`](https://github.com/zalando/postgres-operator/blob/cc1ffdc7b6ed83ef2d1be575db8e4aa76f8c6e57/charts/postgres-operator/values.yaml#L264). But you can remove the name, and the name will be generated by helm. With this creating multiple clusters should work.

If you want the two clusters to not interfere with each other either configure the [watched_namespace](https://github.com/zalando/postgres-operator/blob/cc1ffdc7b6ed83ef2d1be575db8e4aa76f8c6e57/charts/postgres-operator/values.yaml#L112) option or enable setting a [controller ID](https://github.com/zalando/postgres-operator/blob/master/charts/postgres-operator/values.yaml#L292). However this is a very new feature and not available in the `1.4.0` release.
